apps/heft/src/plugins/JestPlugin/JestPlugin.ts

```diff
@@ -145,6 +141,45 @@ export class JestPlugin implements IHeftPlugin {
     clean.properties.pathsToDelete.add(cacheFolder);
   }
 
+  private _getJestReporters(
+    heftSession: HeftSession,
+    heftConfiguration: HeftConfiguration
+  ): (string | Config.ReporterConfig)[] {
+    const config: Config.GlobalConfig = JsonFile.load(this._getJestConfigFolder(heftConfiguration));
+
+    if (Array.isArray(config.reporters)) {
+      // Replace the 'default' reporter with the heft reporter
+      const defaultIndex: number = config.reporters.indexOf('default');
+      if (defaultIndex >= 0) {
```

Comment:
If the default reporter isn't specified, maybe we should print a logging message (maybe a `--verbose` message?) mentioning that we aren't installing Heft's default reporter.